host-interaction/process/allocate-thread-local-storage.yml

```diff
@@ -4,6 +4,11 @@ rule:
     namespace: host-interaction/process
     author: michael.hunhoff@fireeye.com
     scope: function
+    att&ck:
+      - Privilege Escalation::Process Injection::Thread Local Storage [1055.005]
+      - Defense Evasion::Process Injection::Thread Local Storage [1055.005]
```

Comment:
i dont think this strictly indicates process injection. TLS has a valid use within a single process.